test/other/metadce/test_metadce_cxx_noexcept.jssize

```diff
@@ -1 +1 @@
-25948
+26192
```

Comment:
I guess code size is not a big deal for FS-using programs anyhow. It's a shame though that we can't get HandleAllocator small enough to make these PRs code size wins... but I remember your point that once two parts of the code use HandleAllocators then it becomes a savings.